server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -335,6 +337,39 @@ function getContainerName($siteId) {
 	return 'server-contingency-site';
 }
 
+function getToEmailString($siteId) {
+	if($siteId == 1) {
```

Comment:
nitpick: You could actually use `===` here instead. It's the same as in JS, where `===` will check both the type and the value (so `"1" === 1` is false, but `"1" == 1` is true).

Reply:
Changed it! Thanks